### DISCUSSION TITLE
System reinstallation is lost for License

### DISCUSSION THREAD

**Discussion by Raptor3355** [aclap-dev/video-downloadhelper#472] [License & Purchase requests]
### What's your issue?

Hello!

I had to reinstall the system and lost my license.
Do I have to buy the license again?

### The 10 first digits (only 10) of your license number or email address

842a1d73b8

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I sent you an email with your license.
